### ISSUE TITLE
contextual information on searches: command nvda + f3 does not work

### ISSUE THREAD

**Issue by surfer0627** [marlon-sousa/EnhancedFindDialog#4]
Hi,
This issue may caused by NVDA 2021.1 addon API compatibility breaking.

> NVDA what's new: Note: this is an Add-on API compatibility breaking release. Add-ons will need to be re-tested and have their manifest updated. 

### Steps to reproduce:
1. Press NVDA+control+f to find string.
1. Press enter to search.
1. press NVDA+f3 to find next
#### Actual behavior:
NVDA no response.
#### Expected behavior:
Find next works as expected.

#### NVDA installed/portable/running from source:
portable
#### NVDA version:
[2021.1rc2](https://www.nvaccess.org/post/nvda-2021-1rc2/)
#### Windows version:
Windows 10 20H2 (10.0.19042)
#### add-on version:
Enhanced Find Dialog 1.1.0 addon
#### Does the issue still occur after restarting your computer?
Yes.
#### If add-ons are disabled, is your problem still occurring?
No.

#### log message:

```
Input: kb(laptop):NVDA+f3
ERROR - scriptHandler.executeScript (02:27:47.491) - MainThread (5488):
error executing script: <bound method script_enhancedFindNext of <virtualBuffers.gecko_ia2.Gecko_ia2 object at 0x044E60F0>> with gesture 'NVDA+f3'
Traceback (most recent call last):
  File "scriptHandler.pyc", line 208, in executeScript
  File "C:\Users\surfer0627\Downloads\NVDA 2021.1\userConfig\addons\EnhancedFindDialog\globalPlugins\EnhancedFindDialog\cursorManagerHelper.py", line 72, in script_enhancedFindNext
    self.doFindText(self._searchEntries[SEARCH_HISTORY_MOST_RECENT_INDEX], caseSensitive = self._lastCaseSensitivity, searchWrap = self._searchWrap)
  File "C:\Users\surfer0627\Downloads\NVDA 2021.1\userConfig\addons\EnhancedFindDialog\globalPlugins\EnhancedFindDialog\cursorManagerHelper.py", line 92, in doFindText
    speech.speakTextInfo(info,reason=controlTypes.REASON_CARET)
AttributeError: module 'controlTypes' has no attribute 'REASON_CARET'
IO - inputCore.executeGesture (02:27:48.968) - winInputHook (11780):
```

**Comment by surfer0627** [comment]
Thank you for your help and investergating this issue.

**Comment by marlon-sousa** [comment]
Hello, this has now been fixed.

You can find the download at the repository main page.

Thanks,
Marlon

**Comment by surfer0627** [comment]
@marlon-sousa: Thank you for your work.
